rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/executor/BakePoller.groovy

```diff
@@ -174,15 +175,17 @@ class BakePoller implements ApplicationListener<ContextRefreshedEvent> {
       bakeStore.storeBakeError(statusId, errorMessage)
       bakeStore.cancelBakeById(statusId)
 
-      bakesId = registry.createId('bakes').withTag("success", "false").withTag("cause", "failedToUpdateJob")
+      tags = ["success":"false", "cause":"failedToUpdateJob"]
     }
 
-    if (bakesId) {
+    if (!tags.isEmpty()) {
```

Comment:
Any reason to not just use `!tags`? It will return false if there are no mappings.